recovery_backend/server/helper.go

```diff
@@ -0,0 +1,18 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func RespondWithError(rw http.ResponseWriter, code int, message string) {
```

Comment:
Do we have similar things from webhook? If yes, would suggest refactoring a bit then reuse.

Reply:
This is a dead code. Removed it. Thank you.